ApplyToBecomeInternal/ApplyToBecome.Data/Features/ApiClient.cs

```diff
@@ -40,7 +41,22 @@ public async Task<HttpResponseMessage> GetSelectedRegionsAsync(IEnumerable<strin
 
    public async Task<HttpResponseMessage> GetProjectByIdAsync(int id)
    {
-      return await ActiveClient.GetAsync(string.Format(_pathFor.GetProjectById, id));
+      Task<HttpResponseMessage> getProjectResponse = ActiveClient.GetAsync(string.Format(_pathFor.GetProjectById, id));
+      Task<HttpResponseMessage> getProjectNotesResponse =
```

Comment:
Maybe its the time of day but I had to ready through this a few times to figure out what was going on. 
Suggested improvement is to split this out into a couple of methods. But only a nice to have 😁

Reply:
This line just decides whether to go to TRAMS for notes, if the Academisation flag is enabled or not. It can't be split as it's just an if/else. It could be expanded to the more verbose form of that, but I don't think that really adds anything.

If the flag is enabled, this just returns a "no-op" task. If the flag isn't enabled (not using Academisation) then it _actually_ fires a request to the API to load the notes at the same time the request goes to the separate project endpoint.

In TRAMS, they're two separate concepts. In Academisation (inline with the DDD concepts) notes don't exist on their own and are included in the loaded project data.